src/services/bookService.js

```diff
@@ -0,0 +1,15 @@
+const bookDao = require("../models/bookDao");
+
+const getAllBooks = async () => {
+  return await bookDao.getAllBooks();
+};
+
+const getBooksByMain = async id => {
+  const books = await bookDao.getBooksByMain(id);
+  return books;
+};
```

Comment:
해당 부분도 함수명 수정하면 좋을 것 같습니다!

또한 위의 함수처럼 `await bookDao.getBooksByMain(id);` 부분을 바로 return 해도 좋을 것 같습니다.

```suggestion
const getBooksByMain = async id => {
  return await bookDao.getBooksByMain(id);
};
```